Packs/CheckPointDome9/Integrations/CheckPointDome9/CheckPointDome9.py

```diff
@@ -0,0 +1,2194 @@
+
+    return command_results
+
+
+def security_group_tags_update_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """ Update the list of tags for an AWS security group.
+
+    Args:
+        client (Client): Dome9 API client.
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+    """
+    sg_id = args.get('sg_id')
+    key = args.get('key')
+    value = args.get('value')
+
+    response = client.security_group_tags_update_request(sg_id, key, value)
+    readable_output = tableToMarkdown(name='Tag updated successfully',
+                                      t=response,
+                                      headerTransform=string_to_table_header)
+
+    command_results = CommandResults(readable_output=readable_output,
+                                     outputs_prefix='CheckPointDome9.SecurityGroup.Tag',
+                                     outputs_key_field='key',
+                                     outputs=response,
+                                     raw_response=response)
+
+    return command_results
+
+
+def security_group_service_create_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """ Create a new Service (rule) for the security group.
+
+    Args:
+        client (Client): Dome9 API client.
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+    """
+    sg_id = args.get('sg_id')  # mandatory
+    name = args.get('name')  # mandatory
+    protocol_type = args.get('protocol_type')  # mandatory
+    port = arg_to_number(args.get('port'))  # mandatory
+    policy_type = args.get('policy_type')
+    open_for_all = arg_to_boolean(args.get('open_for_all'))
+    description = args.get('description')
+    data_id = args.get('data_id')
+    data_name = args.get('data_name')
+    scope_type = args.get('type')
+    is_valid = args.get('is_valid')
+    inbound = arg_to_boolean(args.get('inbound'))
+    icmptype = args.get('icmptype')
+    icmpv6type = args.get('icmpv6type')
+
+    response = client.security_group_service_create_request(sg_id, name, protocol_type, port,
+                                                            policy_type, open_for_all, description,
+                                                            data_id, data_name, scope_type,
+                                                            is_valid, inbound, icmptype, icmpv6type)
+    sg_service = [{
+        'id': response['id'],
+        'name': response['name'],
+        'protocol Type': response['protocolType'],
+        'port': response['port'],
+        'description': response['description'],
+        'security Group ID': sg_id,
+        #    'scopeType': response['scope'][0]['type'],
+        #    'scopeData': f"{response['scope'][0]['data']['id']} - {response['scope'][0]['data']['name']}",
```

Comment:
what are these commented lines? Should they be deleted?

Reply:
I deleted them:)